massa-execution-worker/src/worker.rs

```diff
@@ -288,8 +288,8 @@ pub fn start_execution_worker(
     // launch the execution thread
     let input_data_clone = input_data.clone();
     let thread_builder = thread::Builder::new()
-        .stack_size(200 * 1024 * 1024)
-        .name("execution".into()); //200 Mb stack size
+        .stack_size(200 * 1024 * 1024) //200 Mb stack size
```

Comment:
Sorry but I forgot to add that maybe a comment about why this size and that it is a temporary workaround would be nice :)